Source/Model/SPNarrowDownCompletion.m

```diff
@@ -785,18 +788,15 @@ - (void)watchUserEvents
 
 	[theView setCompletionIsOpen:YES];
 
-	closeMe = NO;
```

Comment:
Now with additional check added `SPTextView`, `SPNarrowDownCompletion` could be force closed so we don't want to change `closeMe` back to `NO` here.

Note: closeMe is initialized to NO when instance of `SPNarrowDownCompletion` is instantiated anyways so assignment here previously redundant anyways.